graduates-fall-2019.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>New graduates page</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/main.css">
+    <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
+  </head>
+  </head>
+  <body>
+    <header class="main-banner">
+      <nav>
+        <div class="nav-content content-container">
+          <a class="logo-link" href="index.html">
+            <img class="logo" src="img/logo.png" alt="logo">
+          </a>
+
+          <div class="toggle">
+            <i class="fa fa-bars menu" aria-hidden="true"></i>
+          </div>
+          <ul class="main-banner-nav-list">
+            <li><a href="index.html">Home</a></li>
+            <li><a href="about.html">About us</a></li>
+            <li><a href="#volunteer">Volunteer</a></li>
+            <li><a href="graduates-fall-2019.html">Graduates</a></li>
+            <li><a href="#services">Services</a></li>
+            <li><a href="contact.html">Contact us</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <div class="page-template container">
+      <header class="header"></header>
+
+      <main class="figure">
+        <div>
+          <img src="https://via.placeholder.com/150">
+          <p>Graduates from the Fall 2019 Level-2 Refcode class.</p>
+        </div>
+      </main>
```

Comment:
The `<main>` element should contain everything on this page that isn't a header, footer, or aside. So the `.fourColumnLayout` below should be moved up into this `main` element.